flusher.go

```diff
@@ -335,6 +336,28 @@ func (s *Server) reportGlobalMetricsFlushCounts(ms metricsSummary) {
 	s.Statsd.Count(flushTotalMetric, int64(ms.totalTimers), []string{"metric_type:timer"}, 1.0)
 }
 
+func (s *Server) reportGlobalReceivedProtocolMetrics() {
+	protocolMetrics := s.listeningPerProtocolMetrics
+
+	dogstatsdTcpTotal := atomic.SwapInt64(&protocolMetrics.dogstatsdTcpReceivedTotal, 0)
+	dogstatsdUdpTotal := atomic.SwapInt64(&protocolMetrics.dogstatsdUdpReceivedTotal, 0)
+	dogstatsdUnixTotal := atomic.SwapInt64(&protocolMetrics.dogstatsdUnixReceivedTotal, 0)
+	dogstatsdGrpcTotal := atomic.SwapInt64(&protocolMetrics.dogstatsdGrpcReceivedTotal, 0)
+
+	ssfUdpTotal := atomic.SwapInt64(&protocolMetrics.ssfUdpReceivedTotal, 0)
+	ssfUnixTotal := atomic.SwapInt64(&protocolMetrics.ssfUnixReceivedTotal, 0)
+	ssfGrpcTotal := atomic.SwapInt64(&protocolMetrics.ssfGrpcReceivedTotal, 0)
+
+	s.Statsd.Count("listening.received_per_protocol", dogstatsdTcpTotal, []string{"metric_type:global_counter", "veneurglobalonly:true", "protocol:" + DOGSTATSD_TCP.String()}, 1.0)
```

Comment:
Why `metric_type:global_counter`?

Reply:
I copied this from one of the other global metrics being published in flusher.go. I assumed it was a way of capturing in the dimensions that this metric was a global metric of type "counter". I will remove it as it is unlikely being used at this time (as this is a new metric).